shared/api/url.go

```diff
@@ -45,7 +45,7 @@ func (u *URL) Path(pathParts ...string) *URL {
 	}
 
 	u.URL.Path = path.String()
```

Comment:
@simondeziel weird that this line didnt trigger the linter either, as it presumably should also be updated.

Reply:
ah its because of the u.Path() function.